products/BellHybrid/apps/application-bell-settings/widgets/TimeSetListItem.hpp

```diff
@@ -9,12 +9,12 @@
 
 namespace gui
 {
-    class TimeSetFmtSpinner;
+    class TimeSetSpinner;
 
     class TimeSetListItem : public BellSideListItem
     {
       public:
-        TimeSetFmtSpinner *timeSetFmtSpinner = nullptr;
+        TimeSetSpinner *timeSetSpinner = nullptr;
```

Comment:
Use `{}` as you did before in `DateSetSpinner.hpp`

Reply:
done